public/main.js

```diff
@@ -117,4 +124,21 @@ function deleteTodo(id, callback) {
     };
 }
 
+function deleteCompleted() {
+    var completed = [];
+    getTodoList(function(todos) {
+        todos.forEach(function (todo) {
+            if (todo.isComplete) {
+                console.log(todo.id);
+                completed.push(todo.id);
+            }
+        });
+        console.log(completed);
+        completed.forEach(function (id) {
+            deleteTodo(id, function () {});
```

Comment:
I assume this is to be added in a later pull request? If the `function () {}` is just a place holder, its good to put a //TODO comment near it so it's clear that it's not the final code.